<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page1</class>
 <widget class="QMainWindow" name="Page1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>101</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设置数据量大小</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>设置延迟（us）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SizeEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>460</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetSizeButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>460</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>设置大小</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetDelayButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>500</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>设置延迟</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonShuffle">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>460</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>打乱数组</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DelayEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>500</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button4_Bubble">
    <property name="geometry">
     <rect>
      <x>986</x>
      <y>436</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>冒泡排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button2_Binary">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>436</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>二分插入排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button8_Merge">
    <property name="geometry">
     <rect>
      <x>898</x>
      <y>498</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>二路归并排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button7_Heap">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>498</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>堆排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button5_Quick">
    <property name="geometry">
     <rect>
      <x>1074</x>
      <y>436</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>快速排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button6_Selection">
    <property name="geometry">
     <rect>
      <x>722</x>
      <y>498</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button9_Radix">
    <property name="geometry">
     <rect>
      <x>986</x>
      <y>498</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>基数排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3_Shell">
    <property name="geometry">
     <rect>
      <x>898</x>
      <y>436</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>希尔排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button1_Insert">
    <property name="geometry">
     <rect>
      <x>722</x>
      <y>436</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>插入排序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BackButton">
    <property name="geometry">
     <rect>
      <x>1074</x>
      <y>498</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>81</width>
      <height>41</height>
     </size>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_1">
    <property name="title">
     <string>页面1</string>
    </property>
   </widget>
   <addaction name="menu_1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
